source/policies.ttl

```diff
@@ -0,0 +1,17 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix : <https://id.kb.se/vocab/>
+base <https://id.kb.se/>
+
+</policy/freely-available> a :AccessPolicy ;
+    skos:prefLabel "Fritt tillgänglig"@sv, "Freely available"@en .
+
+</policy/gratis> a :AccessPolicy ;
+    skos:prefLabel "gratis"@sv, "gratis"@en ;
+    skos:altLabel "free"@en .
+
+</policy/restricted> a :UsageAndAccessPolicy ;
+    skos:prefLabel "begränsad"@sv, "restricted"@en .
+
+</policy/cc0> a :License ;
```

Comment:
Maybe better to separate policy and license in the uri? (would need a change in the dataset setup as well ofc).

```suggestion
</license/cc0> a :License ;
```

Reply:
Comment out for now with a suggested change to `</term/cc/0>`, i.e. a separate dataset, getting these from creativecommons (or even better, cache records for those when external linked data lands)?